amazon/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM amazonlinux:2
+
+LABEL maintainer="NGINX Controller Engineering"
+
+# e.g '1234567890'
+ARG API_KEY 
+ENV ENV_CONTROLLER_API_KEY=$API_KEY
+
+# e.g https://<fqdn>:8443/1.4
+ARG CONTROLLER_URL
+ENV ENV_CONTROLLER_URL=$CONTROLLER_URL
+
+# e.g True or False
+ARG STORE_UUID=False
+ENV ENV_STORE_UUID=$STORE_UUID
+
+# e.g Instance location already defined in Controller
+ARG LOCATION
+ENV ENV_LOCATION=$LOCATION
+
+# Download certificate (nginx-repo.crt) and key (nginx-repo.key) from the customer portal (https://cs.nginx.com)
+# and copy to the build context
+COPY nginx-repo.* /etc/ssl/nginx/
+COPY nginx-plus-api.conf /etc/nginx/conf.d/
+RUN chmod +r /etc/ssl/nginx/nginx-repo.*
+
+# Install NGINX Plus
+RUN set -ex \
+  && yum -y update && yum -y upgrade \
+  && yum -y groupinstall 'Development Tools' \
+  && yum -y install \
+    ca-certificates \
+    curl \
+    initscripts \
+    sudo \
+    jq \
+    net-tools \
+    python27 \
+    python27-devel \
```

Comment:
Python2.7 dependencies or all of these packages are actually required?

Reply:
that's true - the latest version might not require these packages, but the older one could. We have a separate task for making sure we don't have redundant dependencies, so I'll leave these for now.